server/routes/offenceCodeDecisions/officerDecisionHelper.ts

```diff
@@ -79,17 +85,22 @@ export default class OfficerDecisionHelper extends DecisionHelper {
     const officerData = form.selectedAnswerData as OfficerData
     const searching = !!req.body.searchUser
     if (searching) {
-      const errors = []
+      const searchingErrors = []
       if (!officerData.officerSearchFirstNameInput) {
-        errors.push(error.OFFICER_MISSING_FIRST_NAME_INPUT_SEARCH)
+        searchingErrors.push(error.OFFICER_MISSING_FIRST_NAME_INPUT_SEARCH)
       }
       if (!officerData.officerSearchLastNameInput) {
-        errors.push(error.OFFICER_MISSING_LAST_NAME_INPUT_SEARCH)
+        searchingErrors.push(error.OFFICER_MISSING_LAST_NAME_INPUT_SEARCH)
       }
-      return errors
+      return searchingErrors
     }
     if (!officerData.officerId && !searching) {
-      return [error.OFFICER_MISSING_FIRST_NAME_INPUT_SUBMIT, error.OFFICER_MISSING_LAST_NAME_INPUT_SUBMIT]
+      const submittingErrors = []
+      if (!officerData.officerSearchFirstNameInput) submittingErrors.push(error.OFFICER_MISSING_FIRST_NAME_INPUT_SUBMIT)
+      if (!officerData.officerSearchLastNameInput) submittingErrors.push(error.OFFICER_MISSING_LAST_NAME_INPUT_SUBMIT)
+      if (officerData.officerSearchFirstNameInput && officerData.officerSearchLastNameInput)
+        submittingErrors.push(error.OFFICER_MISSING_ID_SUBMIT)
```

Comment:
Here, and `staffDecisionHelper.ts`, there's a mixture of if statements styles. Not a huge one but folks tend to use if statements with curly braces for consistency and readability